src/azure-cli/azure/cli/command_modules/dls/_client_factory.py

```diff
@@ -26,12 +26,8 @@ def cf_dls_filesystem(cli_ctx, account_name):
     from azure.datalake.store import core
     from azure.cli.core._profile import Profile
 
-    profile = Profile(cli_ctx=cli_ctx)
-    subscription_id = None
-    cred, subscription_id, _ = profile.get_login_credentials(
-        subscription_id=subscription_id,
-        resource=cli_ctx.cloud.endpoints.active_directory_data_lake_resource_id)
+    cred, _, _ = Profile(cli_ctx=cli_ctx).get_login_credentials()
```

Comment:
Please double check if `cli_ctx.cloud.endpoints.active_directory_data_lake_resource_id` needs to be set as `credential_scopes` when creating `AzureDLFileSystem`.